src/api/comms.js

```diff
@@ -91,14 +94,18 @@ export async function request(uri, options = defaultOptions) {
     ...defaultOptions,
     ...options,
     headers
-  }).then(checkStatus);
+  }).then(response => checkStatus(response, stream));
 }
 
-export function get(uri, headers) {
-  return request(uri, {
-    method: 'GET',
-    headers: getHeaders(headers)
-  });
+export function get(uri, headers, stream = false) {
```

Comment:
Could we have an `options` params here so we can name this flag where it's used?

```js
export function get(uri, headers, options = {}) {
```

```diff
const uri = getPodLogURL({ container, name, namespace, follow: 1 });
-return get(uri, { Accept: 'text/plain' }, true);
+return get(uri, { Accept: 'text/plain' }, { stream: true });
```